src/malcolm/malcolmSocketHandler.js

```diff
@@ -21,30 +21,87 @@ const configureMalcolmSocketHandlers = (inputSocketContainer, store) => {
   };
 
   socketContainer.socket.onmessage = event => {
+    console.log(event.data);
```

Comment:
We should probably delete this console statement unless you're still using it for later features.